tests/wdio/secondaryScroll-spec.js

```diff
@@ -0,0 +1 @@
+
```

Comment:
I think this guy can be removed

Reply:
Oops, removed here: https://github.com/cerner/terra-dev-site/pull/217/commits/da753b47dc078f307181297bd340199e3d8f04f5